Modify reclaim dialog to also submit on enter press in input element

Bortseb opened the pull request:
Moved the fetch to /auth/reclaim/ endpoint into form submit event, rather than dialog close event. This makes it so enter press on input element will also submit form, and try to authenticate.

made cancel button close the dialog, which no longer implies trying to authenticate.

WardCunningham reviewed (commented):
Nick and I read through these diffs and it all makes sense.
We did not actually run the code. My only worry would be that there is some difference in the dialog handling between PC and Mac.

Bortseb commented:
I only tested in chrome on Windows... So not sure, but I thought this was all using standardized dialogs/forms from html?

Bortseb commented:
Would mac generally be the same as Linux? Maybe I can set up a virtual environment on my system for better testing? I'll test with Firefox too.

Bortseb commented:
Would mac generally be the same as Linux? Maybe I can set up a virtual environment on my system for better testing? I'll test with Firefox too.

WardCunningham commented:
Robert reports success:
<img width="479" alt="image" src="https://github.com/user-attachments/assets/901b982d-fc2a-47c0-9b42-25c3cd320982" />

Bortseb committed:
Catches syntax errors in the owner.json file

When wiki tries to retrieve the owner for a site, if there is a syntax error in the owner.json, it will crash the whole farm.

This change adds a a bit of graceful degradation. An ill-formed owner file won't crash the farm, and this adds a error notification into the wiki client footer, and also into the wiki logs.
